components/organisms/dark-cards-image.tsx

```diff
@@ -0,0 +1,19 @@
+import { darkCardsImage } from '../../data/dark-cards-image'
+import CardImage from '../molecules/card-image'
+
+export default function DarkCardsImage () {
```

Comment:
Is that for your light and Dark cards for if yes why do you need to have specific color components??